components/account/manage/Delete.js

```diff
@@ -0,0 +1,17 @@
+import Button from "@components/Button";
+import DocumentMinusIcon from "@heroicons/react/24/outline/DocumentMinusIcon";
+
+export default function Delete({ action, id, label = "", children }) {
+  return (
+    <div className="relative w-full">
+      <Button
+        onClick={() => action(id)}
+        aria-label={`Delete ${label}`}
```

Comment:
```suggestion
        aria-label={`Delete ${label}`}
        title={`Delete ${label}`}
```

Hey @eddiejaoude, I think it will be great for UX to have the `title` attribute alongside the screen-reader `aria-label` so we can have a bit of tooltip with user readable content also available.

Reply:
Thank you 👍